utils/database_orchestrator.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+from typing import List
+import os
+
+from utils.sqlite_manager import SQLiteManager
+from utils.os_utils import get_bots_data_paths
+
+
+class DatabaseOrchestrator:
+    def __init__(self):
```

Comment:
Add argument in the constructor and use as default the data folder of the project

Reply:
DatabasesAggregator